orders/src/main/resources/templates/orders.html

```diff
@@ -4,36 +4,69 @@
 <head>
     <meta charset="UTF-8">
     <title>Order</title>
+    <link rel="stylesheet" href="css/orders.css">
 </head>
-<body>
-<H1><span th:text="'Welcome '+${username}+' to the ordering page'"></span></H1>
-<H2>please place your order here</H2>
-
-
-<form action="#" th:action="@{/order}" th:object="${orders}" method="post">
-    <table id="orders_table">
+<body style="font-family:'David',sans-serif">
+<div style="text-align: center;">
+    <H1><span th:text="${username}+' ברוך הבא '"></span></H1>
+    <H2>אנא הוסף את המוצרים הרצויים להזמנה</H2>
+    <table class="tg" id="orderTable">
         <thead>
         <tr>
-            <th>מוצר</th>
-            <th>ברקוד</th>
-            <th>מחיר ליחידה</th>
-            <th>כמות</th>
+            <th class="tg-cscf">מחלקה</th>
+            <th class="tg-cscf">קוד מוצר</th>
+            <th class="tg-cscf">שם מוצר</th>
+            <th class="tg-cscf">מחיר ליחידה/100 גרם</th>
+            <th class="tg-cscf">כמות להזמנה</th>
+            <th class="tg-cscf">סה"כ מחיר</th>
         </tr>
         </thead>
         <tbody>
-        <tr th:each="order, itemStat : *{orders}">
-            <input type="hidden" th:field="*{orders[__${itemStat.index}__].name}">
-            <input type="hidden" th:field="*{orders[__${itemStat.index}__].barcode}">
-            <input type="hidden" th:field="*{orders[__${itemStat.index}__].price}">
-            <td th:text="*{orders[__${itemStat.index}__].name}"></td>
-            <td th:text="*{orders[__${itemStat.index}__].barcode}"></td>
-            <td th:text="*{orders[__${itemStat.index}__].price}"></td>
-            <td><input th:field="*{orders[__${itemStat.index}__].quantity}"></td>
+        <tr>
+            <td class="tg-0pky">
+                <label for="selectDepartment"></label><select id="selectDepartment" onchange="departmentSelected()">
+                <option>בחר מחלקה...</option>
+            </select>
+            </td>
+            <td class="tg-0pky"><label for="enterBarcode"></label><input type="text" id="enterBarcode"
```

Comment:
as mentioned, I believe that in this time, it's not correct to allow typing a barcode.
Also, is "onchange" happening every time I type a letter? seems like a source of problems if it is the case

Reply:
You cannot enter a barcode - the input is disabled. The name and function are there for future implementation - the function is not complete and fully tested, but has the basic functionallity we'll need. Then, once we'll have the radio button - we'll just enable the barcode and disable the name.